README.md

```diff
@@ -35,7 +42,7 @@ pyenv install 3.9.x
 
 Clone the repository
 ```
-git clone git@github.com:HewlettPackard/py-spiffe.git
+Follow through the steps in [Contributing](#Contributing) session.
```

Comment:
I think this should be `section` instead of `session`

Reply:
Done, thanks :)